Chinese/monk/alibaba/alibaba.md

```diff
@@ -0,0 +1 @@
+很久以前，在波斯的一座城市里住着两兄弟：哥哥卡西姆娶了一个有钱的老婆，生活非常富有；弟弟阿里巴巴夫妻俩靠打柴为生，过着穷苦的生活。
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.